src/data_structures/mod.rs

```diff
@@ -1,5 +1,6 @@
+pub mod array_2d;
 pub mod k_mer_array;
 pub mod k_mer_multi;
-pub mod tree;
+pub mod suffix_trees;
 pub mod transcriptome;
-pub mod suffix_trees;
\ No newline at end of file
+pub mod tree;
```

Comment:
this looks like the "Optimize Imports" option in IntelliJ, I'll look for it in VSCode, or a way to run "rustfmt" on every save.